### PR TITLE
PRL-6986: Change content on case creation screens

### PR DESCRIPTION
**Pull request by hcarslaw**
### Jira link
https://tools.hmcts.net/jira/browse/PRL-6986

### Change description
Updates content on case creation screens 

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by sealDebolina — APPROVED** [review]

File: src/main/steps/edge-case/type-of-application/routeGuard.ts [hunk 1]
```diff
@@ -11,6 +12,21 @@ export const routeGuard = {
         edgeCaseTypeOfApplication: req.body.edgeCaseTypeOfApplication,
       } as CaseWithId;
```
Comment (sealDebolina) [inline src/main/steps/edge-case/type-of-application/routeGuard.ts:14]: not include function with FGM, FMPO might be better as we are planning to add more C1(CA type ) application.that might reduce line of coding.

**Review by sealDebolina — CHANGES_REQUESTED** [review]

File: src/main/steps/edge-case/type-of-application/routeGuard.ts (hunk 1 above)
Comment (hcarslaw) [reply src/main/steps/edge-case/type-of-application/routeGuard.ts:15]: Added a function to check for FGM or FMPO and used it here to check if FGM or FMPO are not the case type.
